package.json

```diff
@@ -46,7 +48,28 @@
     "transformIgnorePatterns": [
       "node_modules/(?!(@guardian/src-foundations|@guardian/types)/)"
     ],
-    "snapshotSerializers": ["jest-emotion"],
-    "testPathIgnorePatterns": ["dist"]
+    "snapshotSerializers": [
+      "jest-emotion"
+    ],
+    "testPathIgnorePatterns": [
+      "dist"
+    ],
+    "globals": {
+      "ts-jest": {
+        "babelConfig": {
+          "presets": [
+            [
+              "@babel/preset-env",
+              {
+                "targets": {
+                  "node": "10"
```

Comment:
Can this be linked to `.nvmrc` in any way?

Reply:
Yeah we can create an `.nvmrc`. I've targeted Node 10 because we run the tests for 10 and 12 - as DCR is still on 10.